src/ansible_navigator/actions/lint.py

```diff
@@ -0,0 +1,343 @@
+
+
+def severity_to_color(severity: str) -> int:
+    """Convert severity to curses colors."""
+    if severity == "minor":
+        return 5
+    if severity == "major":
+        return 3
+    if severity in ("critical", "blocker"):
+        return 1
+    if severity == "info":
+        return 6
+    return 0
+
+
+def color_menu(colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
+    # pylint: disable=unused-argument
+    """Color the menu."""
+    return (severity_to_color(entry["severity"]), 0)
+
+
+def content_heading(obj: Dict, screen_w: int) -> Union[CursesLines, None]:
+    """Generate the content heading."""
+    check_name = obj["check_name"]
+    check_name = check_name + (" " * (screen_w - len(check_name)))
+    path_line = f"PATH: {abs_user_path(obj['location']['path'])}"
+    check_name_line = f"MESSAGE: {check_name}"
+
+    return (
+        (
+            CursesLinePart(
+                column=0,
+                string=path_line,
+                color=0,
+                decoration=curses.A_BOLD,
+            ),
+        ),
+        (
+            CursesLinePart(
+                column=0,
+                string=check_name_line,
+                color=severity_to_color(obj["severity"]),
+                decoration=curses.A_UNDERLINE,
+            ),
+        ),
+    )
+
+
+def filter_content_keys(obj: Dict[Any, Any]) -> Dict[Any, Any]:
+    """when showing content, filter out some keys"""
+    ignored_keys = ("fingerprint",)
+    return {k: v for k, v in obj.items() if not k.startswith("__") and k not in ignored_keys}
+
+
+def massage_issues(issues: List[Dict]) -> List[Dict]:
+    """Massage issues by injecting some useful keys with strings for rendering."""
+    out = []
+    for issue in issues:
+        issue["__message"] = issue["check_name"].split("] ", 1)[1].capitalize()
+        issue["__path"] = abs_user_path(issue["location"]["path"])
+        if isinstance(issue["location"]["lines"]["begin"], Mapping):
+            issue["__line"] = issue["location"]["lines"]["begin"]
+        else:
+            issue["__line"] = issue["location"]["lines"]["begin"]
+        issue["issue_path"] = f"{issue['__path']}:{issue['__line']}"
+        out.append(issue)
+    return out
+
+
+@actions.register
+class Action(App):
+    """:lint"""
+
+    KEGEX = r"^lint(\s(?P<params>.*))?$"
+
+    def __init__(self, args: ApplicationConfiguration):
+        super().__init__(args=args, logger_name=__name__, name="lint")
+
+    @property
+    def is_interactive(self):
+        """are we interactive?"""
+        return self._args.mode == "interactive"
+
+    def _fatal(self, msg: str, rc: int = 1) -> Tuple[str, str, int]:
+        # pylint: disable=invalid-name
+        self._logger.error(msg)
+
+        if self.is_interactive:
+            notification = error_notification(messages=[msg])
+            self._interaction.ui.show(notification)
+        else:
+            raise RuntimeError(msg)
+
+        return "", "", rc
```

Comment:
Does this really need to return a tuple? I only saw one place where the return value is even used.
Also, `rc` is passed into this method and never used, just returned back to the caller — why can't the caller just use it?